### PR TITLE
Branch1

### PR DESCRIPTION
**Pull request by MartinPetrakiev**

### CODE REVIEW

**Review by mvlt — APPROVED** [review]

File: Compare_arrays/Program.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            int[] arr = new int[n];
+
+            for (int i = 0; i < n; i++)
+                arr[i] = Convert.ToInt32(Console.ReadLine());
+
+            return arr;
+        }
+        static string CompareElementsOfTwoArrays(int[] arrayOne, int[] arrayTwo, int size)
+        {
+            string output = "Equal";
```
Comment (mvlt) [inline Compare_arrays/Program.cs:18]: Do you really need last parameter "size"

**Review by mvlt — CHANGES_REQUESTED** [review]

File: Remove_elements_from_array/Program.cs [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+            return maxLength;
+        }
+
+        static void Main(string[] args)
+        {
+            int n = Convert.ToInt32(Console.ReadLine());
+
+            int[] myArray = InitializeInputIntArray(n);
+
+            FindMInimalRemovalsToSort(myArray, n);
```
Comment (mvlt) [inline Remove_elements_from_array/Program.cs:62]: Why do you need second param?

File: Remove_elements_from_array/Program.cs [hunk 3]
```diff
@@ -0,0 +1,82 @@
+﻿using System;
+
+namespace MyProject
+{
+    class Program
+    {
+        static int[] InitializeInputIntArray(int n)
```
Comment (mvlt) [inline Remove_elements_from_array/Program.cs:7]: Seems more like populating array from input

**Review by mvlt — CHANGES_REQUESTED** [review]
Other changes look fine

File: Remove_elements_from_array/Program.cs (hunk 2 above)
Comment (MartinPetrakiev) [reply Remove_elements_from_array/Program.cs]: [135a85b](https://github.com/MartinPetrakiev/c_sharp_arrays/pull/1/commits/135a85bb0ff7fb5aa954d7a595347d285a9f0baf)

**Pull request merged**